test/functional/testCases.js

```diff
@@ -2289,18 +2373,564 @@ const testCases = [
+                }
+            }
+        ]
+    },
+    // 0700 - ENTITY NAME EXPRESSION TESTS
+    {
+        describeName: '0700 - Group JEXL entityNameExpression',
+        provision: {
+            url: 'http://localhost:' + config.iota.server.port + '/iot/services',
+            method: 'POST',
+            json: {
+                services: [
+                    {
+                        resource: '/iot/json',
+                        apikey: globalEnv.apikey,
+                        entity_type: globalEnv.entity_type,
+                        entityNameExp: 'type+":"+(t*2*static_a)',
+                        commands: [],
+                        lazy: [],
+                        attributes: [
+                            {
+                                object_id: 's',
+                                name: 'status',
+                                type: 'Boolean'
+                            },
+                            {
+                                object_id: 't',
+                                name: 'temperature',
+                                type: 'Number'
+                            }
+                        ],
+                        static_attributes: [
+                            {
+                                name: 'static_a',
+                                type: 'Number',
+                                value: 3
+                            }
+                        ]
+                    }
+                ]
+            },
+            headers: {
+                'fiware-service': globalEnv.service,
+                'fiware-servicepath': globalEnv.servicePath
+            }
+        },
+        should: [
+            {
+                shouldName:
+                    'A - WHEN sending provisioned object_ids (measures) through http IT should store the entity with the expression generated ID into the Context Broker',
+                measure: {
+                    url: 'http://localhost:' + config.http.port + '/iot/json',
+                    method: 'POST',
+                    qs: {
+                        i: globalEnv.deviceId,
+                        k: globalEnv.apikey
+                    },
+                    json: {
+                        s: false,
+                        t: 10
+                    }
+                },
+                expectation: {
+                    id: 'TestType:60',
+                    type: globalEnv.entity_type,
+                    status: {
+                        value: false,
+                        type: 'Boolean'
+                    },
+                    temperature: {
+                        value: 10,
+                        type: 'Number'
+                    },
+                    static_a: {
+                        value: 3,
+                        type: 'Number'
+                    }
+                }
+            }
+        ]
+    },
+    // 0700 - FULL GROUP TESTS
+    // APIKEY RECOGNITION IS NOT WORKING FOR LIB, SKIPPED
```

Comment:
Placeholder for multigroups tests. Not working in lib, probably it should work at agent level. NTC